dubbo-dependencies-bom/pom.xml

```diff
@@ -106,7 +106,7 @@
         <consul_process_version>2.0.0</consul_process_version>
         <xmemcached_version>1.3.6</xmemcached_version>
         <cxf_version>3.1.15</cxf_version>
-        <thrift_version>0.8.0</thrift_version>
+        <thrift_version>0.11.0</thrift_version>
```

Comment:
pls. bump up version to the latest. see this: https://github.com/dubbo/rpc-native-thrift-for-apache-dubbo/network/alert/pom.xml/org.apache.thrift:libthrift/open